integration-libs/cpq-quote/cpq-quote-discount/components/cpq-quote-discount-tbody/cpq-quote.component.html

```diff
@@ -0,0 +1,31 @@
+<td class="center" *ngIf="!isFlagquote">
```

Comment:
Plase, rename all your classes then the class should start with `cx-` prefix and be easy to understand